cmd/storage-rest-client.go

```diff
@@ -287,7 +287,10 @@ func (client *storageRESTClient) DiskInfo(_ context.Context) (info DiskInfo, err
 			var info DiskInfo
 			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 			defer cancel()
-			respBody, err := client.call(ctx, storageRESTMethodDiskInfo, nil, nil, -1)
+
+			vals := make(url.Values)
+			vals.Set(storageRESTMetrics, strconv.FormatBool(metrics))
```

Comment:
@harshavardhana The first call then decides (forever) if metrics are fetched. I will fix this for the grid implementation.

Reply:
not forever right, it will be for 1sec @klauspost

Reply:
@harshavardhana It is captured in the `sync.Once` closure, so whatever it was then, it will remain that.